client/my-sites/checkout/composite-checkout/hooks/use-create-payment-methods/use-create-existing-cards.ts

```diff
@@ -20,15 +20,26 @@ export default function useCreateExistingCards( {
 	// card processor does require it (for 3DS cards), so we wait to create the
 	// payment methods until stripe is loaded.
 	const shouldLoad = ! isStripeLoading && ! stripeLoadingError;
+
 	// Memoize the cards by comparing their stored_details_id values, in case the
 	// objects themselves are recreated on each render.
-	const memoizedStoredCards: StoredCard[] = useMemoCompare( storedCards, ( prev, next ) => {
-		const prevIds = prev?.map( ( card ) => card.stored_details_id ) ?? [];
-		const nextIds = next?.map( ( card ) => card.stored_details_id ) ?? [];
-		return (
-			prevIds.length === nextIds.length && prevIds.every( ( id, index ) => id === nextIds[ index ] )
-		);
-	} );
+	const memoizedStoredCards: StoredCard[] | undefined = useMemoCompare(
```

Comment:
I think you can throw away these changes for now, as we are no longer relying on the tax info on `StoredCard` objects.

Reply:
I've removed these changes.